posts/2023-07-09_falcon/falcon.qmd

```diff
@@ -23,15 +23,15 @@ long_slug <- "2023-07-09_falcon"
 
 ## What is {falcon}?
 
-The [{falcon} initiative](https://pharmaverse.github.io/falcon/) is an industry collaborative effort under [pharmaverse](https://pharmaverse.org/) that unites Boehringer Ingelheim, Moderna, Roche, and Sanofi with the aspiration of building and open-sourcing a comprehensive catalog of harmonized tables, listings, and graphs (TLGs) for clinical study reporting. By leveraging existing open-source R packages, {falcon} aims to simplify the process of output review, comparison, and meta-analyses, while fostering efficient communication among stakeholders in the pharmaceutical industry.
+The [{falcon} initiative](https://pharmaverse.github.io/cardinal/) is an industry collaborative effort under [pharmaverse](https://pharmaverse.org/) that unites Boehringer Ingelheim, Moderna, Roche, and Sanofi with the aspiration of building and open-sourcing a comprehensive catalog of harmonized tables, listings, and graphs (TLGs) for clinical study reporting. By leveraging existing open-source R packages, {falcon} aims to simplify the process of output review, comparison, and meta-analyses, while fostering efficient communication among stakeholders in the pharmaceutical industry.
 
 ## Why do we build it?
 
 The collaborative effort focuses on improving the clarity, consistency, and accessibility of TLGs by addressing variations and redundancies in their creation and use. This harmonized approach allows for streamlined reporting processes and facilitates effective communication of study results within the industry and to regulatory authorities.
 
 ## What has been done so far?
 
-Drawing inspiration from the FDA Standard Safety Tables and Figures Integrated Guide, the {falcon} initiative has successfully developed and open-sourced 11 templates to date. 4 product owners and 11 developers from 4 companies have collaborated to make these templates available and also published them on the official {falcon} website at [https://pharmaverse.github.io/falcon/](https://pharmaverse.github.io/falcon/).
+Drawing inspiration from the FDA Standard Safety Tables and Figures Integrated Guide, the {falcon} initiative has successfully developed and open-sourced 11 templates to date. 4 product owners and 11 developers from 4 companies have collaborated to make these templates available and also published them on the official [{falcon} website](https://pharmaverse.github.io/cardinal/).
 
 ## Next steps & vision
 
```

Comment:
Hi, Just out of curiousity.  This issue #188 is for the Coursera blog post. I'm unsure how the 'falcon.qmd' post should be a part of this issue.  Should the facon updates be its own issue?

Reply:
I added it here because the links triggered the CICD pipelines. 
But ideally you are right, should be its own issue and then merged into main and then merged into this branch :)